src/spyglass/lfp_band/lfp_band_merge.py

```diff
@@ -0,0 +1,40 @@
+import datajoint as dj
+import pandas as pd
+
+from spyglass.common.common_nwbfile import AnalysisNwbfile
+from spyglass.lfp_band.v1.lfp_band import LFPBandV1  # noqa F401
+from spyglass.utils.dj_helper_fn import fetch_nwb
+
+schema = dj.schema("lfp_band_merge")
+
+
+@schema
+class LFPBandOutput(dj.Manual):
+    definition = """
+    lfp_band_id: uuid
+    ---
+    source: varchar(40)
+    version: int
+
+    """
+
+    class LFPBandV1(dj.Part):
+        definition = """
+        -> LFPBandOutput
+        -> LFPBandV1
+        ---
+        -> AnalysisNwbfile
+        lfp_band_object_id: varchar(40)
+        """
+
+        def fetch_nwb(self, *attrs, **kwargs):
```

Comment:
I wonder if `fetch_nwb` could be a mixin class to save us the boiler-plate

Reply:
Definitely something I've thought about but haven't had time to do. Feel free to add it to the list.